<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pigeonclient_main</class>
 <widget class="QMainWindow" name="pigeonclient_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="teMessageBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMessage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>691</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwUsers">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>171</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
